sdk/textanalytics/test-resources.json

```diff
@@ -50,6 +50,7 @@
         }
     },
     "variables": {
+        "roleDefinitionId": "[concat('/subscriptions/', subscription().subscriptionId, '/providers/Microsoft.Authorization/roleDefinitions/a97b65f3-24c7-4388-baec-2e87135dc908')]",
```

Comment:
Is this a hardcoded ID because it has to be enabled by text analytics for the feature flagging?

Reply:
Not sure I understand what you mean. Could you expand more?

Reply:
I am following what other pipelines are doing though:
- Form Recognizer: https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/formrecognizer/test-resources.json#L82
- App Config: https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/appconfiguration/test-resources.json#L57
- TA on Java: https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/textanalytics/test-resources.json#L45

Let me know if this is not the right approach.

Reply:
This ID corresponds to the `Cognitive Services User` role (more info [here](https://docs.microsoft.com/en-us/azure/role-based-access-control/built-in-roles)). We're assigning this role to our service principal as a requirement to use AAD.

I think we may be able to use the role name as well, but not sure.